spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java

```diff
@@ -152,6 +152,21 @@ public void open() {
       }
     }
 
+    String remoteRepo = getProperty("zeppelin.interpreter.remoteRepo");
```

Comment:
Could you elaborate why you're trying to add `zeppelin.interpreter.remoteRepo` into the classpath?
As far as i understand, value in the property would be something like `http://repo1.maven.org/maven2/`.